### ISSUE TITLE
feat: Keyboard buttons for media control

### ISSUE THREAD

**Issue by miggi92** [music-assistant/companion#34]
Would be nice if I could use the keyboard buttons for "pause", "resume", "skip", "reverse" for controling the current media.

I'm currently using the companion with a windows 11 laptop.

**Comment by arctixdev** [comment]
Yeah actually I have been looking into it. We would simply have to utilize the [MediaSessionAPI](https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API), but that requires playing audio in the webview which we currently arent doing since its using Squeezelite. This could be circumvented by just playing empty audio... At the moment I am very busy with exams coming up but I might be able to implement it later... Otherwise anyone else is also welcome to pick up the issue
